packages/modal/test/Modal.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { render } from 'react-testing-library'
+import { Modal } from '../src/index'
+import { ThemeProvider, theme } from 'pcln-design-system'
+
+const customRender = (node, ...options) => {
```

Comment:
Should this customRender function be a common utility since you use it across a couple files.  Also, could you clarify it's purpose?  Not sure how it's being used.

Reply:
@sdalonzo might have a better idea since it's part of react-testing-lib